scipy/stats/_continuous_distns.py

```diff
@@ -8854,6 +8854,206 @@ def _updated_ctor_param(self):
+           https://en.wikipedia.org/wiki/Studentized_range_distribution
+    .. [2] Batista, Ben Dêivide, et al. "Externally Studentized Normal Midrange
+           Distribution." Ciência e Agrotecnologia, vol. 41, no. 4, 2017, pp.
+           378–389., doi:10.1590/1413-70542017414047716.
+    .. [3] Harter, H. Leon. “Tables of Range and Studentized Range.” The Annals
+           of Mathematical Statistics, vol. 31, no. 4, 1960, pp. 1122–1147.
+           JSTOR, www.jstor.org/stable/2237810. Accessed 18 Feb. 2021.
+    .. [4] Lund, R. E., and J. R. Lund. “Algorithm AS 190: Probabilities and
+           Upper Quantiles for the Studentized Range.” Journal of the Royal
+           Statistical Society. Series C (Applied Statistics), vol. 32, no. 2,
+           1983, pp. 204–210. JSTOR, www.jstor.org/stable/2347300. Accessed 18
+           Feb. 2021.
+
+    Examples
+    --------
+    >>> from scipy.stats import %(name)s
+    >>> import matplotlib.pyplot as plt
+    >>> fig, ax = plt.subplots(1, 1)
+
+    Calculate a few first moments:
+
+    %(set_vals_stmt)s
+    >>> mean, var, skew, kurt = %(name)s.stats(%(shapes)s, moments='mvsk')
+
+    Display the probability density function (``pdf``):
+
+    >>> x = np.linspace(%(name)s.ppf(0.01, %(shapes)s),
+    ...                 %(name)s.ppf(0.99, %(shapes)s), 100)
+    >>> ax.plot(x, %(name)s.pdf(x, %(shapes)s),
+    ...        'r-', lw=5, alpha=0.6, label='%(name)s pdf')
+
+    Alternatively, the distribution object can be called (as a function)
+    to fix the shape, location and scale parameters. This returns a "frozen"
+    RV object holding the given parameters fixed.
+
+    Freeze the distribution and display the frozen ``pdf``:
+
+    >>> rv = %(name)s(%(shapes)s)
+    >>> ax.plot(x, rv.pdf(x), 'k-', lw=2, label='frozen pdf')
+
+    Check accuracy of ``cdf`` and ``ppf``:
+
+    >>> vals = %(name)s.ppf([0.001, 0.5, 0.999], %(shapes)s)
+    >>> np.allclose([0.001, 0.5, 0.999], %(name)s.cdf(vals, %(shapes)s))
+    True
+
+    Rather than using (``%(name)s.rvs``) to generate random variates, which is
+    very slow for this distribution, we can approximate the inverse CDF using
+    an interpolator, and then perform inverse transform sampling with this
+    approximate inverse CDF.
+
+    Using the shape parameters, obtain the bounds data that will be input to
+    ``cdf`` for later use in interpolation.
+
+    >>> a, b = %(name)s.support(k, v)
+    >>> a, b
+    (0, inf)
+
+    Note that the upper limit for the range for the ``cdf`` input is
+    technically infinite. Inputting 1 to the ``ppf`` gives infinity:
+
+    >>> %(name)s.ppf(1, k, v)
+    inf
+
+    To obtain a finite limit we can input a value that nears 1.
+
+    >>> b = %(name)s.ppf(.999, k, v)
+    >>> b
+    7.41058083802274
+
+    >>> from scipy.interpolate import interp1d
+    >>> xs = np.linspace(a, b, 50)
+    >>> cdf = %(name)s.cdf(xs, k, v)
+    >>> ppf = interp1d(cdf, xs, kind='next', fill_value='extrapolate')
```

Comment:
```suggestion
    # Create an interpolant of the inverse CDF
    >>> ppf = interp1d(cdf, xs, fill_value='extrapolate')
```
`kind='next'` was appropriate for a discrete distribution, but not here.